src/redux/sagas/authSaga.ts

```diff
@@ -52,36 +62,45 @@ function postSignUp(body: object): Promise<boolean> {
 function* login(
   action: IAuthActionCreateLogin
 ): Generator<CallEffect | PutEffect> {
-  try {
-    const isLoggedIn = yield call(postLogin, action.payload);
-    yield put({ type: AuthTypes.SET_IS_LOGGED_IN, isLoggedIn });
-    if (isLoggedIn) {
-      yield call(Router.push, "/");
-    }
-  } catch (e) {
-    yield put({ type: AuthTypes.SET_IS_LOGGED_IN, isLoggedIn: false });
+  const isLoggedIn = yield call(postLogin, action.payload);
+  yield put({ type: AuthTypes.SET_IS_LOGGED_IN, isLoggedIn });
+  // Redirect to landing page if login is successful
+  if (isLoggedIn) {
+    yield call(Router.push, "/");
   }
 }
 
 function* signUp(
   action: IAuthActionCreateSignUp
 ): Generator<CallEffect | PutEffect> {
   const payload = action.payload;
-  const body = {
+  const body: IPostSignUpBody = {
+    csrfToken: payload.csrfToken,
```

Comment:
We don't need send the CSRF as part of the sign up so you don't make this change.
```suggestion
```

Reply:
Updated to remove this